Raspi/Test/mock_wireless_module.py

```diff
@@ -44,6 +47,8 @@
                    ("longitude", 25),
                    ("altitude", 50),
                    ("course", 0))
+s_power = MockSensor(200, percent_range=0.8)
+s_cadence = MockSensor(90, percent_range=0.20)
```

Comment:
I will add these to my PR as I've removed a lot of the hard coded elements that you have added.

Reply:
The values are really good though